models/tiktok_ads__ad_report.sql

```diff
@@ -1,81 +1,83 @@
-
 with hourly as (
     
     select *
     from {{ var('ad_report_hourly') }}
 
-), campaigns as (
+), ads as (
 
     select *
-    from {{ ref('int_tiktok_ads__most_recent_campaign') }}
+    from {{ ref('int_tiktok_ads__most_recent_ad') }}
 
-), advertiser as (
+), ad_groups as (
 
     select *
-    from {{ var('advertiser') }}
+    from {{ ref('int_tiktok_ads__most_recent_ad_group') }}
```

Comment:
Juts looked at Google and Microsoft, I see what you mean. The reason they're intermediate models was because they're reused and referenced across different end models. Which I think typically should be best practice but in this case I think reducing computation > reusing makes much more sense since there's not much going on in the intermediate models here anyways. I'll try doing it the more straightforward way!